linera-client/src/client_options.rs

```diff
@@ -92,6 +92,13 @@ pub struct ClientContextOptions {
     #[arg(long, default_value = "10")]
     pub max_retries: u32,
 
+    /// OpenTelemetry exporter type for distributed tracing.
+    /// Valid values: "chrome_json", "otlp" (requires tempo feature and
+    /// will be used as the default).
+    /// Can also be set via OTEL_EXPORTER_TYPE environment variable.
+    #[arg(long, env = "OTEL_EXPORTER_TYPE")]
```

Comment:
We should be preppending all env vars with `LINERA_` and we've already started doing that in a couple of places.